---
layout: page
title: CV
permalink: /cv/
---

---

<div class="title-container">
  
  <div class="name-and-links">
    <h2>Naraen Palanikumar</h2>
    <p class="social-links">
      <a href="mailto:naraen.p@gmail.com">
        <img src="{{ '/assets/icons/mail.svg' | relative_url }}" alt="Mail Icon">
        Email
      </a> |
      <a href="https://www.linkedin.com/in/naraenp/">
        <img src="{{ '/assets/icons/linkedin.svg' | relative_url }}" alt="LinkedIn Icon">
        LinkedIn
      </a> |
      <a href="https://github.com/naraenp">
        <img src="{{ '/assets/icons/github.svg' | relative_url }}" alt="GitHub Icon"> 
        GitHub
      </a> |   
      <a href="https://orcid.org/0000-0001-8513-4590">
        <img src="{{ '/assets/icons/orcid.svg' | relative_url }}" alt="OrcID Icon">
        OrcID
      </a>
    </p>
  </div>

  <a href="/assets/pdf/naraenp_resume.pdf" class="download-button" download>
    Download PDF
  </a>

</div>

### <mark>Summary</mark>
Highly motivated **Bioengineering** graduate student with a strong background in **computational biology**, **data analysis**, and **interdisciplinary research**. Proficient in applying and developing bioinformatics and statistical pipelines for complex biological datasets, including **high-throughput sequencing** and **physiological data**. Experienced in utilizing **Python**, **R**, **cloud computing platforms**, and **high-performance computing environments**. Proven ability to contribute to **research publications**, **develop analytical tools**, and **collaborate effectively** within scientific teams. Eager to leverage expertise in **bioengineering**, **data science**, and **research methodologies** to drive innovation.

### <mark>Education</mark>
**University of Illinois at Urbana-Champaign**
* **Master of Engineering (M.Eng.), Bioengineering** \
*(Aug 2023 - May 2024 | Champaign, IL)*
* **Bachelor of Science (B.S.), Bioengineering** \
*(Aug 2019 - May 2023 | Champaign, IL)*

### <mark>Experience</mark>
**Bioinformatics Research Assistant** \
Malloy Lab, Uniformed Services University of the Health Sciences \
*(Oct 2024 - Jun 2025 | Bethesda, MD)*
* Investigated respiratory infectious diseases and their progression through advanced bioinformatic analyses, leading to improved understanding of disease mechanisms. 
* Processed incoming biological samples for flow cytometry and single-cell RNA sequencing immediately upon arrival, ensuring high-quality data for subsequent analyses. 
* Contributed to the analysis and interpretation of large-scale transcriptomic datasets, preparing findings for publication.

**Healthcare Technician** \
Carle Foundation Hospital Emergency Department, Carle Health \
*(Sep 2023 - Jul 2024 | Champaign, IL)*
* Performed critical patient care tasks, including EKGs, blood collection, and urinalysis.
* Facilitated seamless information flow between patients and providers in a fast-paced environment. 
* Managed patient requests and assisted in various emergency department sections, from triage to specialized EKG technician roles. 

**Undergraduate Research Assistant** \
Sweeney Lab, University of Illinois at Urbana-Champaign \
*(Aug 2021 - Jun 2023 | Champaign, IL)* 
* Utilized R and Python to analyze single-cell RNA sequencing (scRNAseq) and fiber photometry data, transforming raw data into actionable biological insights. 
* Developed an interactive analytical application for fiber photometry data, significantly enhancing lab efficiency and data processing capabilities. 
* Co-authored a peer-reviewed publication in The Journal of Neuroscience, demonstrating strong data interpretation and scientific communication skills. 

**Emergency Medical Technician (EMT-B)** \
Illini Emergency Medical Services, University of Illinois at Urbana-Champaign \
*(Jan 2022 - May 2023 | Champaign, IL)* 
* Provided immediate medical response and direct patient care at university athletic and cultural events, ensuring smooth operation and expedient care. 
* Maintained meticulous records of incidents and managed essential medical equipment.

**Undergraduate Research Assistant** \
Thom Lab, Wheaton College \
*(May 2020 - Aug 2020 | Wheaton, IL)* 
* Conducted a meta-analysis of 54 studies using the R programming language, involving data extraction, transformation, and statistical analysis. 
* Developed a comprehensive seven-part video series to document analytical methods and workflows, ensuring reproducibility and facilitating training for future lab members. 

### <mark>Technical Skills</mark>
* **Programming Languages:** R, Python, Shell Scripting, SQL 
* **Bioinformatics & Data Analysis:** High-Throughput Sequencing Data Analysis, scRNAseq, Physiological Data Collection & Analysis, Statistical Modeling, Algorithm Development, Data Transformation & Engineering, Data Visualization, Metadata Capture, Version Control, Machine Learning, Deep Learning
* **Computing Environment:** UNIX/Linux, Mac, Windows, High-Performance Computing (HPC) systems, 10x Genomics platform, Google Cloud platform (GCP) 
* **Bioinformatics Tools & Workflows:** Seurat, Cellrank, 10x Genomics Pipeline, Open-Source Bioinformatics Tools, Pandas, NumPy, SciPy, PyTorch
* **Laboratory Skills:** Biological Sample Processing, PCR, Flow Cytometry 
* **Software:** MS Office Suite, Google Suite, VS Code, R Studio

### <mark>Awards & Recognition</mark>
* **National AP Scholar:** *(May 2019)*
* **National Merit Scholarship Finalist:** *(February 2019)*
* **USA Biology Olympiad Semifinalist:** *(April 2018)*

### <mark>Associated Publications</mark>
1. **Title:** Paraventricular Thalamic MC3R Circuits Link Energy Homeostasis with Anxiety-Related Behavior \
  a. **Authors:** *Dajin Cho, Kyle O'Berry, Ingrid Camila Possa-Paranhos, Jared Butts, Naraen Palanikumar, and Patrick Sweeney.* \
  b. **Date of Publication:** *(Sep 2023)* \
  c. This paper elucidates a novel neurobiological pathway in which a specific population of neurons acts as a crucial link between energy regulation and anxiety. These findings offer a new understanding of how metabolic states can directly influence behavior, with significant implications for both obesity and psychiatric research. My primary contribution to this paper was the development of an interactive analytical application using R with an R Shiny external interface. This tool was used to more efficiently process and analyze fiber photometry data, which transformed raw data into actionable biological insights through figures presented in the publication.

2. **Title:** Patterns of restricted TCR usage following SARS-CoV-2 vaccination and severe disease \
  a. **Authors:** *Emily Parsons, Zhongyan Lu, Stephanie A. Richards, Amanda Zelkoski, Janifer Le, Naraen Palanikumar, Phuong Nguyen, Allison M. W. Malloy, et al.* \
  b. **Date of Publication:** *(Aug 2025)* \
  c. We currently have a publication in review submitted to the journal Frontiers in Immunology that examines the differences in T-cell receptor expression between participants who demonstrate more serious symptoms of COVID-19 versus those who did not. These differences are also analyzed with respect to participant vaccination status. My role within this paper was to complete a bioinformatic analysis of the CD4 T-cells and graphically represent their distribution amongst the participants.

### <mark>Languages</mark>
* **English:** *(Fluent)*
* **Spanish:** *(Professional)*

### <mark>Exam Scores</mark>

| **Exam Name** | **Score** | **Percentile** | **Date** |
| :--- | :--- | :--- | :--- |
| **MCAT** | 521 / 528 | 98th | Jan 2024 |
| **ACT** | 35 / 36 | 99th | Jul 2018 |
| **SAT** | 1550 / 1600 | 99th | Apr 2018 |

---